src/com/company/Main.java

```diff
@@ -8,18 +8,40 @@
 
 
 public class Main {
-    public static void main(String[] args){
+    public static void main(String[] args) {
+
+        String txtFilePath = args[0];
+
+        File file = new File(txtFilePath);
```

Comment:
What if the user did not provide any arguments?

Reply:
it prints the the error message as "Please enter the valid path and word" and terminate the execution.